pkg/cvo/cvo.go

```diff
@@ -139,6 +139,10 @@ type Operator struct {
 	// lastAtLock guards access to controller memory about the sync loop
 	lastAtLock          sync.Mutex
 	lastResourceVersion int64
+
+	// excludeManifestRegularExpressions is a set of regular expresions that are used
+	// to exclude manifest file names that match them.
+	excludeManifestRegularExpressions []string
```

Comment:
why not store regex compiled here? compiling these regex every time is known to cause problems.